addons/GME/Scripts/Game/GME/Mission/SCR_MissionHeader.c

```diff
@@ -0,0 +1,6 @@
+//------------------------------------------------------------------------------------------------
+modded class SCR_MissionHeader : MissionHeader
+{
+	[Attribute(defvalue: "NONE", desc: "Global AI skill", uiwidget: UIWidgets.ComboBox, enums: ParamEnumArray.FromEnum(EAISkill))]
+	EAISkill m_eGME_GlobalAISkill;
```

Comment:
Is the default skill level value loaded in somewhere else, as here it is set as "NONE"? 

Or is the logic flow that a "NONE" means no changes to the AI skill as I see you do not set the skill on new AI units if it is equal to NONE?

Reply:
Yeah, `NONE` means that the skill of new AI units should not be changed from its default.

Reply:
Is it possible to set it back to ``NONE`` after it have been changed once? I can't quite find it here on github if the UI allows to set it back to ``NONE`` after having changed it once. 

And if it does, will the logic then be able to actually set it back to "default", as changing to ``NONE`` then returns early, and the existing units are stuck in non-default, but whatever we applied last? 

I am away from home so can't test it in workbench.

Reply:
It is possible to go back to `NONE` (see `GME_BaseAISkillEditorAttribute.FillValues` and overrides in derived classes). Doing so will not change the skill of already placed entities at the moment, but future spawned entities are left untouched. So yeah, they are stuck with what we applied last.

Reply:
I guess having it reset to the default value would be more sensible though

Reply:
If feasible I think it would be good to change them back to the default if GM sets it back to ``NONE``. 

Alternative we could decide ``NONE`` is not an option, so whenever the module have been applied, you are stuck in our scale of AI settings. Not as user-friendly, but also not as confusing from a user perspective, of what ``NONE`` means and how does the AI settings by default get set etc.

Reply:
Resolved in d690eee